src/components/Organization/Dashboard/ProcessCard.tsx

```diff
@@ -11,13 +16,76 @@ const ProcessCard = () => {
 
   return (
-    <Link as={RouterLink} to={`/processes/${ensure0x(election.id)}`} w='full'>
+    <Link as={RouterLink} to={`/processes/${ensure0x(election.id)}`} w='full' textDecoration='none'>
       <ContentsBox w='full'>
-        <ElectionTitle fontSize='md' textAlign='left' />
-        <ElectionSchedule fontSize='xs' textAlign='left' />
+        <Flex
+          flexDirection={{ base: 'column', lg: 'row' }}
+          alignItems={{ base: 'start', lg: 'center' }}
+          justifyContent='space-between'
+        >
+          <ElectionStatusBadge fontSize='text-sm' mb={{ base: 3, lg: 0 }} order={{ lg: '2' }} />
+          <ElectionTitle fontSize='text' textAlign='left' fontWeight='extrabold' order={{ lg: '1' }} />
+        </Flex>
+        <HStack mb={3}>
+          <MdHowToVote />
+          <Text fontWeight='600' fontSize='text-sm' color='dashboard_card_text' className='dashborad_align_card_text'>
+            {(election as any).voteCount} ({((election as any).voteCount / (election as any).census.size) * 100}%) votes
+            submited
+          </Text>
+        </HStack>
+        <HStack>
+          <IoIosCalendar />
+          <HStack>
+            <Text fontWeight='600' fontSize='text-sm' className='dashborad_align_card_text'>
+              <Text as='span' color='gray'>
+                {(election as any).startDate.getTime() < new Date().getTime() ? (
+                  <Trans i18nKey='dashboard.card.started'>Started</Trans>
+                ) : (
+                  <Trans i18nKey='dashboard.card.starts'>Starts</Trans>
+                )}
+                :
+              </Text>{' '}
+              <Text as='span' color='dashboard_card_text'>
+                {formatDate((election as any).startDate)}
+              </Text>
+            </Text>
+
+            <IoIosArrowRoundForward />
+            <Text fontWeight='600' fontSize='text-sm' className='dashborad_align_card_text'>
+              <Text as='span' color='gray'>
+                {(election as any).endDate.getTime() < new Date().getTime() ? (
+                  <Trans i18nKey='dashboard.card.finished'>Finished</Trans>
+                ) : (
+                  <Trans i18nKey='dashboard.card.finish'>Finish</Trans>
+                )}
+                :
+              </Text>{' '}
+              <Text as='span' color='dashboard_card_text'>
+                {formatDate((election as any).endDate)}
+              </Text>
+            </Text>
+          </HStack>
+        </HStack>
       </ContentsBox>
     </Link>
   )
 }
+const locales = {
+  en: enUS,
+  es: es,
+  ca: ca,
+}
+
+const formatDate = (date: Date) => {
+  const { i18n } = useTranslation()
+
+  const locale = locales[i18n.language as keyof typeof locales] || enUS
 
+  const formattedDate = format(date, 'MMM dd, yyyy, HH:mm', { locale })
+
+  const [monthDay, year, time] = formattedDate.split(', ')
+  const [month, day] = monthDay.split(' ')
+
+  return `${month} ${day} - ${year} (${time})`
+}
```

Comment:
Why?
1. We have methods to do this. We actually have a hook to format dates `const {format} = useDateFns()` will return the function you want, with all the locales stuff done.
2. This is not the place for a format date function, at all.